### DISCUSSION TITLE
CP detecting the wrong working width

### DISCUSSION THREAD

**Discussion by crcarmichael** [Courseplay/courseplay#7712] [Q&A]
Where does CP pull the working width from in the course generation screen?  Some mods I have always have the wrong width.  I've looked in the id3 and xml files and the widths look correct. Any ideas on how I could edit the problematic mods to work well with CP?

**Comment by Tensuko** [comment]
Just change it in generator and fine. Or use the vehicle configuration.

**Reply by crcarmichael** [reply]
It's just a pain clicking the width up to the correct number all the times. Not sure what you mean by vehicle configuration, though.

**Reply by Tensuko** [reply]
why not use the scroll wheel ?
